internal/stores/hostdb.go

```diff
@@ -358,7 +363,7 @@ func (db *SQLHostDB) Hosts(notSince time.Time, max int) ([]hostdb.Host, error) {
 		Joins("JOIN host_interactions ON host_interactions.Host = hosts.Public_Key").
 		Select("Public_key").
 		Group("Public_Key").
-		Having("MAX(Timestamp) < ?", notSince).
+		Having("MAX(Timestamp) < ?", notSince.UTC()). // use UTC since we stored timestamps in UTC
```

Comment:
ok *this* makes it very clear to me why we should be storing timestamps in the DB as UTC

Reply:
kinda sucks that we have to be careful to add `UTC` on every `time.Time` value though. Easy to forget.